cmd/import/import.go

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
For internal consistency &ndash;as discussed via Slack&ndash; I'd rename `cmd` by `cli`

Reply:
done in https://github.com/src-d/code-annotation/pull/10/commits/20e8e3f13e90268221a1b05f4cf1fd42cbc4d4b2